config/initializers/sidekiq.rb

```diff
@@ -6,14 +6,25 @@ def self.redis_url
 
```

Comment:
Adding middleware per: https://github.com/mhenrixon/sidekiq-unique-jobs#add-the-middleware